.github/workflows/pipeline.yml

```diff
@@ -11,19 +11,21 @@ on:
 
 jobs:
   # old-school build and jar method. No tests run or compiled.
-  build-1_6: 
+  build-8: 
```

Comment:
This whole `build-8` section still needs to be deleted. All it does is waste time in the pipeline.

Reply:
lines 13-41 should be deleted.